src/components/Comments/ReplyCommentItem.js

```diff
@@ -222,7 +304,7 @@ const ReplyCommentItem = ({
             )}
           </ButtonHightlight>
         </TouchableOpacity>
-        <Text style={styles.vote(voteStyle())}>{totalVote}</Text>
+        {totalVote !== 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition `totalVote !== 0` has been added to the rendering of the vote count. This change might cause confusion for users when they see no vote count at all instead of a zero. It's generally better UX to show a zero count rather than hiding the count entirely.
```diff
-  {totalVote !== 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
+  <Text style={styles.vote(voteStyle())}>{totalVote}</Text>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->